### PR TITLE
Handle events

### PR DESCRIPTION
**Pull request by bc-bbay**

### CODE REVIEW

**Comment by gkatsev** [comment]
LGTM but looks like it needs a rebase.

File: src/js/player.js [hunk 1]
```diff
@@ -1739,19 +1740,20 @@ Player.prototype.onUserInactive;
...
-Player.prototype.onVolumeChange;
+Player.prototype.handleVolumeChange;
 
 /**
  * Fired when an error occurs
  * @event error
  */
-Player.prototype.onError;
+Player.prototype.handleError;
+<<<<<<< HEAD
```
Comment (heff) [inline src/js/player.js]: There's some merge issues happening here.

Comment (heff) [inline src/js/player.js]: run `git mergetool`?

**Comment by heff** [comment]
Other than the merge issue, looks good to me.

**Pull request closed without merging**